test/verifiedTests/testTools/testCalcGroupStats.m

```diff
@@ -8,11 +8,11 @@
 %     - Lemmer El Assal February 2017
 %
 
-% save the current path
-currentDir = pwd;
+% define the path to The COBRAToolbox
+pth = which('initCobraToolbox.m');
+CBTDIR = pth(1:end-(length('initCobraToolbox.m') + 1));
 
-% initialize the test
-initTest(fileparts(which(mfilename)));
+cd([CBTDIR, filesep, 'test', filesep, 'verifiedTests', filesep, 'testTools']);
```

Comment:
@lemmerelassal , may you change this back to the original version? It has been recently changed with PR #324.